src/amber/controller/helpers/responders.cr

```diff
@@ -51,8 +61,9 @@ module Amber::Controller::Helpers
 
     private def requested_responses
       req_responses = Array(String).new
-
-      if (accept = context.request.headers["Accept"]?) && !accept.empty?
+      if (ext = request.path.match(Content::REGEX).try(&.[1]))
```

Comment:
What about using `.last` instead of `&.[1]` since is more readable?

Reply:
A regex match returns a hash instead of an array, 1 is just a key for the first capture group. Similar to $1 in perl.

Reply:
Does it break with `request.path.match(Content::REGEX)[1]?` ?